packages/expo-modules-autolinking/scripts/ios/xcode_env_generator.rb

```diff
@@ -0,0 +1,13 @@
+require 'pathname'
+
+
+def maybe_generate_xcode_env_file!()
+  project_directory = Pod::Config.instance.project_root
+  xcode_env_file = File.join(project_directory, '.xcode.env.local')
+  if File.exists?(xcode_env_file)
+    return
+  end
+  node_path = `command -v node`.strip!
```

Comment:
Check `$CHILD_STATUS` to see if this fails. Also if `node_path` is empty for any reason, we should probably not generate the env file.